### DISCUSSION TITLE
Acesso à dados com .NET, C#, Dapper e SQL Server (40%) > Dapper > Primeira Consulta

### DISCUSSION THREAD

**Discussion by DrenokBR** [balta-io/forum#437] [Tira Dúvidas] [answered]
Fala André ! Boa tarde !

Cara, fiquei com uma dúvida sobre o Mapeamento De/Para do Dapper.
A dúvida é a seguinte: E se a TABELA do DB e as CLASSES da Aplicação tiverem tamanhos diferentes... ?!?! 

Por exemplo... e se a Tabela tiver mais COLUNAS do que a quantidade de PROPRIEDADES da CLASSE na hora de mapear ? O que acontece ? O Dapper alimenta as propriedades correspondentes e ignora as outras COLUNAS da TABELA do DB ?!?!?

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @DrenokBR, como vai?

Neste caso as colunas são ignoradas. O mesmo vale para as propriedades... se tiverem mais propriedades do que colunas, elas vem com o valor padrão.

**Reply by DrenokBR** [reply]
Maravilha !!!

Cara, estou muito satisfeito de ter adquirido o acesso a sua página. Melhor coisa que Eu fiz. Tu é muito bom na didática, no domínio e na produção do conteúdo !!! Além de parecer ser um cara muito gente boa.

E muito obrigado pela atenção !

**Reply by andrebaltieri** [reply]
Bom dia, @DrenokBR! Muito bom ler isto. Fico feliz demais!
Vamos pra cima 🚀
